src/DIRAC/Core/Tornado/Server/private/BaseRequestHandler.py

```diff
@@ -674,7 +674,12 @@ def on_finish(self):
         argsString = f"OK {self._status_code}"
         # Finish with DIRAC result
         if isReturnStructure(self.__result):
-            argsString = "OK" if self.__result["OK"] else f"ERROR: {self.__result['Message']}"
+            if self.__result["OK"]:
+                argsString = "OK"
+            else:
+                argsString = f"ERROR: {self.__result['Message']}"
+                if callStack := self.__result.pop("CallStack", None):
+                    argsString += "\n" + "".join(callStack)
```

Comment:
FYI: This is completely different to what was needed for v7r3 so it's worth someone else actually checking it